pkgs/top-level/all-packages.nix

```diff
@@ -13723,7 +13709,8 @@ let
     retroarch = retroarchBare;
   });
 
-  rssglx = callPackage ../misc/screensavers/rss-glx { };
+  rss-glx = callPackage ../misc/screensavers/rss-glx { };
+  rssglx = builtins.trace "WARNING: rssglx package was renamed to rss-glx" pkgs.rss-glx;
```

Comment:
Better to add this to https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-unstable.xml